src/main/java/com/checkmarx/flow/custom/GitHubIssueTracker.java

```diff
@@ -205,6 +225,37 @@ private void handleIssueUpdateError(HttpClientErrorException e) {
         }
     }
 
+    private void addCommentToAnUpdatedIssue(String issueUrl, IssueStatus issueStatus) {
+        StringBuilder commentFormat = setIssueUpdatedDescription(issueStatus);
+        this.addComment(issueUrl, commentFormat.toString());
```

Comment:
is it commentFormat or the comment itself?

Reply:
addComment() gets the commentFormat as a string to execute the final comment on the issue

Reply:
When I see xxxFormat, I expect a format string, something with %s etc.